src/content/docs/journey-large-logs/organize.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Organize your logs 
+metaDescription: "Test"
+---
+
+import logsParsing from 'images/logs_screenshot_full-parsing.webp'
+import logsPartition from 'images/logs_screenshot_full-partition.webp'
+
+Once you've decided what logs to ingest and store, it's time to organize your logs. You're still likely ingesting hundreds of gigabytes or dozens of terabytes of logs. While it's a lot less than you originally had, you'll still have toil while trying to effectively use them. 
+
+New Relic provides the ability to group these remaining logs into thematic partitions and parse the logs to pull out and tag valuable attributes. By organizing your logs in this manner, you can:
+
+* Query for any attribute within your logs
+* Manage specific partitions at a time, such as front-end logs vs back-end logs
+* Reduce query load times 
+
+
+## Why partition logs
+
+Seperating your logs into discrete partitions provides both performance benefits and management benefits. For example, imagine you want to query for an attribute only found in front-end logs. Querying through 5 TB of logs it will take drastically longer than querying through a specific 1 TB partition of only front-end logs. 
```

Comment:
`provides both performance benefits and management benefits`

Two things:

* You probably don't need to repeat `benefits`
* But, better still, I think you can use verbs and get moer specific. e.g. `... can improve performance and make your logs easier to manage`.